torch/csrc/lazy/python/init.cpp

```diff
@@ -1,12 +1,607 @@
 #include <torch/csrc/lazy/python/init.h>
 
+#include <torch/csrc/jit/python/pybind.h>
 #include <torch/csrc/lazy/core/debug_util.h>
+#include <torch/csrc/lazy/core/metrics.h>
 #include <torch/csrc/lazy/python/python_util.h>
 
+
+// #include <c10/core/Device.h>
```

Comment:
Any reasons why we commit so many lines of commented out code?

Reply:
this isn't going to be landed this way.  I just wanted to bikeshed the layout of the files and the module conventions, then start to complete the bindings

also, sorry that this PR shows all the changes from the previous 2 diffs in the stack.  Hopefully you only looked at the changes from the top commit.